h2o-algos/testMultiNode.sh

```diff
@@ -21,11 +21,11 @@ case "`uname`" in
 esac
 
 function cleanup () {
-  kill -9 ${PID_1} ${PID_2} ${PID_3} ${PID_4} 1> /dev/null 2>&1
+  kill -9 ${PID_11} ${PID_21} ${PID_31} ${PID_41} ${PID_51} 1> /dev/null 2>&1
   wait 1> /dev/null 2>&1
-  RC=`cat $OUTDIR/status.0`
-  if [ $RC -ne 0 ]; then
-    cat $OUTDIR/out.0
+  RC="`paste $OUTDIR/status.* | sed 's/\s*//g'`"
+  if [ $RC != "00000" ]; then
```

Comment:
I am not sure if `exit $RC` will work properly with `exit 00000` on MacOSX.

From: http://tldp.org/LDP/abs/html/exit-status.html

> Every command returns an exit status (sometimes referred to as a return status or exit code). A successful command returns a 0, while an unsuccessful one returns a non-zero value that usually can be interpreted as an error code. Well-behaved UNIX commands, programs, and utilities return a 0 exit code upon successful completion, though there are some exceptions.